rust/ql/test/library-tests/modules/main.rs

```diff
@@ -0,0 +1,201 @@
+mod my; // I1
```

Comment:
Perhaps the directory name `modules` -> `path-resolution` would make it more clear what is being tested?